### ISSUE TITLE
Incorrect proxy request syntax

### ISSUE THREAD

**Issue by nikkictl** [sensu/sensu-docs#1069]
https://docs.sensu.io/sensu-go/5.1/reference/checks/#proxy-requests-attributes

## This is an issue with:

- [ ] Bug (site functionality or styling)
- [x] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)

## Expected Behavior
```
"entity_attributes": ["entity.entity_class == 'proxy'"]
```

## Current Behavior
```
"entity_attributes": ["entity.EntityClass == 'proxy'"]
```

## Possible Solution
Simple fix for the example.

## Steps to Reproduce (for bugs)
1. https://docs.sensu.io/sensu-go/5.1/reference/checks/#proxy-requests-attributes

## Context
Community user attempted to use `entity.EntityClass` rather than `entity.entity_class`.

* Browser(s) and version(s): Safari
* OS: MacOS

**Comment by cwjohnston** [comment]
Closed by #1066

**Comment by apaskulin** [comment]
Thanks, @nikkiki! Looks like we just got this one with #1066 :)
